td-agent/msi/parameters.wxi.erb

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Include>
-  <?define VersionNumber="<%= version %>" ?>
-  <?define DisplayVersionNumber="<%= version %>" ?>
+  <?define VersionNumber="<%= wix_package_version %>" ?>
+  <?define DisplayVersionNumber="<%= wix_package_version %>" ?>

```

Comment:
There is no need to unify to wix_package_version for DisplayVesionNumber.
I mean "DisplayVersionNumber == version", VersionNumber ==  wix_package_version.

Reply:
:+1: